docs/spec/api.md

```diff
@@ -2814,3 +3026,103 @@ The error codes that may be included in the response body are enumerated below:
 
 
 
+### Catalog
+
+List a set of available repositories in the local registry cluster. Does not provide any indication of what may be available upstream. Applications can only determine if a repository is available but not if it is not available.
+
+
+
+#### GET Catalog
+
+Retrieve a sorted, json list of repositories available in the registry.
```

Comment:
On Tue, Jun 02, 2015 at 10:13:56AM -0700, Richard Scothern wrote:
> > +Retrieve a sorted, json list of repositories available in the registry.
> 
> Is the entire list sorted, or each paginated section?  If it's the
> former does that mean you need to fetch the entire tag repository
> list from storage in order to dispatch paginated sections?

I expect the pagination and sorting requirements are just going to get
passed through (with an altered storage API).